fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/metric/GenericMetricLoader.java

```diff
@@ -62,26 +60,17 @@ public void loadMetricDictionary(MetricDictionary metricDictionary) {
                 .forEach(dataSourceConfiguration -> {
                     dataSourceConfiguration.getMetrics()
                             .forEach(metric -> {
-                                ApiMetricName apiMetricName = new FiliApiMetricName(
-                                        metric,
-                                        dataSourceConfiguration.getValidTimeGrain()
+                                metrics.add(
+                                        new MetricInstance(
+                                                new LogicalMetricInfo(metric),
+                                                doubleSumMaker,
+                                                new DruidMetricName(metric)
+                                        )
                                 );
-                                FieldName fieldName = new DruidMetricName(metric);
-                                metrics.add(new MetricInstance(apiMetricName, doubleSumMaker, fieldName));
                             });
                 });
 
+        Utils.addToMetricDictionary(metricDictionary, metrics);
```

Comment:
This is not a new method. It's the old `addToMetricDictionary ` that's been moved to Util class.